addons/website_theme_install/models/theme_models.py

```diff
@@ -143,6 +143,7 @@ def _convert_to_base_model(self, website, **kwargs):
 
 class Theme(models.AbstractModel):
     _name = 'theme.utils'
+    _inherit = 'web_editor.assets'
```

Comment:
Could be an idea but no... can't you just **use** the `web_editor.assets` model in theme utils like you do it in the controller ?

Reply:
so with `request.env['web_editor.assets']` ?

Reply:
`request` is for controllers. You are in a model here, so you can simply use `self.env[...]`